qiita_pet/handlers/api_proxy/sample_template.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import division
+import warnings
+
+from os import remove
+from os.path import exists, join
+
+from natsort import natsorted
+# This is the only folder in qiita_pet that should import from outside
+# qiita_pet. The idea is this proxies the call and response dicts we expect
+# from the Qiita API once we build it. This will be removed and replaced with
+#  API calls when the API is complete.
+from pandas.parser import CParserError
+from qiita_db.metadata_template.sample_template import SampleTemplate
+from qiita_db.study import Study
+from qiita_core.util import execute_as_transaction
+
+from qiita_db.metadata_template.util import (load_template_to_dataframe,
+                                             looks_like_qiime_mapping_file)
+from qiita_db.util import get_mountpoint
+from qiita_db.exceptions import (QiitaDBColumnError, QiitaDBExecutionError,
+                                 QiitaDBDuplicateError, QiitaDBError,
+                                 QiitaDBDuplicateHeaderError)
+from qiita_ware.metadata_pipeline import (
+    create_templates_from_qiime_mapping_file)
+from qiita_ware.exceptions import QiitaWareError
+from qiita_pet.util import convert_text_html
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def sample_template_info(samp_id, user_id):
```

Comment:
Could you expand on what you mean by it makes it easier to search? Find works regardless of the string being searched for.